chainio/clients/avsregistry/writer.go

```diff
@@ -254,7 +254,7 @@ func (w *AvsRegistryChainWriter) RegisterOperatorInQuorumWithAVSRegistryCoordina
 	if err != nil {
 		return nil, errors.New("failed to send tx with err: " + err.Error())
 	}
-	w.logger.Infof("tx hash: %s", tx.Hash().String())
+	w.logger.Info("tx succesfully included", "tx hash", receipt.TxHash.String())
 	w.logger.Info("registered operator with the AVS's registry coordinator")
```

Comment:
I would combine those two log records into a single record that includes all the data.

Reply:
Done: [bdd6003](https://github.com/Layr-Labs/eigensdk-go/pull/114/commits/bdd60037541edc1deadf63d1ca1f2240dd4ade06)